tests/base.py

```diff
@@ -127,17 +127,20 @@ def expected_metadata(self):
                 self.PRIMARY_KEYS: {"ID"},
                 self.REPLICATION_METHOD: self.INCREMENTAL,
                 self.REPLICATION_KEYS: {"ModifiedDate"},
-            } 
+            }
         }
 
-    def streams_to_select(self):
+    def streams_to_select(self, is_discovery=False):
```

Comment:
Defining this logic in base in an anti-pattern. We should not abstract away the streams under test. It should be defined at the test level.